types/shared.d.ts

```diff
@@ -367,14 +367,18 @@ export interface StripeConnectWrapper {
 
 type ConnectHTMLElementRecordBase = {
   [tagName in ConnectElementTagName]: HTMLElement;
-}
+};
 
 interface ConnectHTMLElementRecord extends ConnectHTMLElementRecordBase {
-  'account-onboarding': HTMLElement & {
+  "account-onboarding": HTMLElement & {
     setFullTermsOfServiceUrl: (termOfServiceUrl: string | undefined) => void;
-    setRecipientTermsOfServiceUrl: (recipientTermsOfServiceUrl: string | undefined) => void;
+    setRecipientTermsOfServiceUrl: (
+      recipientTermsOfServiceUrl: string | undefined
+    ) => void;
     setPrivacyPolicyUrl: (privacyPolicyUrl: string | undefined) => void;
-    setSkipTermsOfServiceCollection: (skipTermsOfServiceCollection: boolean | undefined) => void;
+    setSkipTermsOfServiceCollection: (
+      skipTermsOfServiceCollection: boolean | undefined
```

Comment:
@kaiying FYI the `setter` methods in `submerchant-surfaces` currently always take in a string, not a boolean. This would require some changes in `submerchant-surfaces` to support both `string` and `boolean`. I have some ideas on how we could do that easily, lmk if you want to discuss!

Reply:
Curious why can't we just pass boolean there, since [this](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/frontend/connect/submerchant-surfaces/src/connect/utils/wrapComponentAsCustomElement.ts#L55-59) doesn't really enforce the type?

Reply:
discussed offline, this should be fine!